client/test/test_client.py

```diff
@@ -22,12 +24,21 @@ def test_client(self):
         api_client = client.Client("http://localhost:8002", "superuser", "superuser")
 
         # Read routes
-        self._test_route_return(api_client.get_my_devices(), list)
+        all_devices = self._test_route_return(api_client.get_my_devices(), list)
         self._test_route_return(api_client.get_sites(), list)
         self._test_route_return(api_client.get_all_alerts(), list)
         self._test_route_return(api_client.get_ongoing_alerts(), list)
         self._test_route_return(api_client.get_unacknowledged_alerts(), list)
 
+        if len(all_devices) > 0:
+            # Create event
+            event_id = self._test_route_return(api_client.create_event(0., 0.), dict)['id']
+            # Create an alert
+            alert = self._test_route_return(api_client.send_alert(0., 0., event_id, all_devices[0]['id']), dict)
+            # Acknowledge it
+            updated_alert = self._test_route_return(api_client.acknowledge_alert(alert['id']), dict)
```

Comment:
considering the error I pointed above, I am surprised this test pass

Reply:
agreed, I'm surprised as well, but it did pass locally for me :man_shrugging: